simulations/CC_cycle.py

```diff
@@ -247,14 +258,14 @@ def output(solution, an, sep, ca, params, sim, plot_flag=True,
     n_plots = 2 + an.n_plots + ca.n_plots + sep.n_plots
 
     # There are 4 variables stored before the state variables: (1) time (s),
-    # (2) cycle number, (3) current density(A/cm2) , and (4) Capacity (mAh/cm2)
-    SV_offset = 4
+    # (2) cycle number, (3) current density(A/cm2) ,  (4) Capacity (mAh/cm2) and (5) power density and (6) energy density
+    SV_offset = 6
```

Comment:
Depending how we answer above, either keep this as is (if we store cumulative densities), or re-set to 4.